com.unity.render-pipelines.high-definition/Documentation~/HDRP-Features.md

```diff
@@ -263,13 +263,19 @@ In HDRP, you set up clouds inside a [Volume](Volumes.md), so you can change clou
 In HDRP, you set up fog, inside a [Volume](Volumes.md), so you can change fog settings, or even the fog type itself, depending on the position of the Camera in the Scene. You can set the color of the fog yourself or use the color of the sky. HDRP fog affects Materials with both opaque and transparent surfaces. HDRP implements an exponential [fog](Override-Fog.md) with optional volumetric effects.
 
 
+### Local Volumetric Fog
+
+![](Images/LocalVolumetricFog.png)
+
+In addition to fog, HDRP supports local volumetric fog which helps you control the density of the fog in an area. To allow more fine grained control, it's also possible to use a 3D Mask texture to control the color and the density inside the volume itself. For more information on the Local Volumetric Fog volume, see the [Local Volumetric Fog section](Local-Volumetric-Fog.md).
```

Comment:
In addition to fog, HDRP also supports local volumetric fog. You can use this to control the density of fog in an area. For more detailed control, you can use a 3D Mask texture to control the color and the density inside the volume itself. For more information see the [Local Volumetric Fog](Local-Volumetric-Fog.md) documentation.